src/backends/github/API.js

```diff
@@ -19,6 +19,10 @@ export default class API {
     return this.request("/user");
   }
 
+  collaborator(user) {
```

Comment:
This method should be renamed and modified as per my refactoring suggestions below.